exercises/nth-prime/example.lua

```diff
@@ -0,0 +1,18 @@
+return function(n)
+  assert(n > 0, 'n must be positive')
+
+  local primes = {}
+
+  local function is_prime(n)
+    for _, p in ipairs(primes) do
+      if n % p == 0 then return false end
+      if p^2 > n then return true end
```

Comment:
It would be interesting to include a test that does profiling of some sort, not sure if other tracks do that but the go track has benchmark tests